RF24.cpp

```diff
@@ -9,6 +9,8 @@
 #include "nRF24L01.h"
 #include "RF24_config.h"
 #include "RF24.h"
+#include <stdlib.h>
+#include <unistd.h>
```

Comment:
I can't test this right now, but are these header files compatible with Arduino?

Reply:
Hmm. I have to check it!

Reply:
In Arduino Nano RF24_SPI_TRANSACTIONS is not set so the geteuid and exit functions will not be included.

Object dump confirms that both geteuid and exit functions are included in Raspberry PI but not in Arduino.

So it does not matter if stdlib.h and unistd are compatible because the functions used are not actually included in code! That means that the linker does not link these two functions in Arduino elf.

I believe that it wont hurt if these headers are included in Arduino program.

Reply:
Also tested in real Arduino program. 
No errors thrown!